src/main/java/com/micetr0/mock_DB/Mock_DB.java

```diff
@@ -191,12 +191,12 @@ public void insertAccount(Account account)
      * @return List of Account ID's that have the specified username
      */
     @Override
-    public List<Integer> findAccountIdByUsername(String username) {
-        List<Integer> accountIdList = new ArrayList<>();
+    public List<Account> findAccountByUsername(String username) {
+        List<Account> accountIdList = new ArrayList<>();
```

Comment:
Rename to accountList, since it is a list of accounts.

Reply:
done